tests/car_tests.rs

```diff
@@ -99,36 +102,45 @@ async fn forest_cli_car_concat_same_file_3_times() -> Result<()> {
 
 async fn new_car(size: usize, path: impl AsRef<Path>) -> Result<()> {
     let rng = SmallRng::seed_from_u64(0xdeadbeef);
-    let (cid, _data) = new_block(&mut rng.clone());
-    let header = CarHeader::from(vec![cid]);
-
-    let mut block_stream = Box::pin(
-        futures::stream::unfold(rng, |mut rng| async { Some((new_block(&mut rng), rng)) })
-            .take(size),
+    let root_block = new_block(&mut rng.clone());
+    let roots = vec![root_block.cid];
+    let block_stream = futures::stream::iter(vec![Ok(root_block)]).chain(
```

Comment:
I don't understand this change. Doesn't this give you two copies of the `root_block`?

Reply:
Hmm I did not notice `rng.clone()` did the trick to make sure `root_block` is rerenerated, reverted